### PR TITLE
No language requirement displays twice TM-1382

### PR DESCRIPTION
**Pull request by scott062**

### CODE REVIEW

**Comment by scott062** [comment]
Should I do I do a separate PR to remove this for projected vacancies as well?

**Comment by rtirserio** [comment]
@scott062 you can do them both in one PR

**Comment by scott062** [comment]
I updated both in the filter reducer so that the correct NLR option displays for available positions and projected vacancies when the feature flags are true. However, this also removed the NLR option completely if you flip those flags false. Is there any reason we care about the NLR option displaying when the available positions and/or pv flags are false?

**Comment by mjoyce91** [comment]
@scott062 we need NLR to display if `available_positions` is false.

File: src/reducers/filters/filters.js [hunk 1]
```diff
@@ -1,17 +1,19 @@
...
     const hasAltData = has(m, 'item.dataAP');
+    const hasInitDataAP = has(m, 'item.initialDataAP');
     return {
       ...m,
       item: {
         ...m.item,
         endpoint: useAP && hasAPEndpoint ? m.item.endpointAP : m.item.endpoint,
       },
       data: useAP && hasAltData ? m.dataAP : m.data,
+      initialData: useAP && hasInitDataAP ? m.initialDataAP : m.initialData,
```
Comment (mjoyce91) [inline src/reducers/filters/filters.js]: Was this still needed with the logic being moved to the action?

**Comment by scott062** [comment]
So I finally got it fixed so that it shows the original added language field with AP flag OFF, and it shows the new API 'No Language Requirement' when AP flag ON. @mjoyce91 noticed that the PV filter may not be updating correctly for the NLR option so I will look into that some more (perhaps on second PR).

File: src/reducers/filters/filters.js (hunk 1 above)
Comment (scott062) [reply src/reducers/filters/filters.js]: Nope. Thanks! Just changed it

**Comment by mjoyce91** [comment]
@scott062 check merge conflicts

**Pull request merged**